manual/source/Tutorial/DataStructures/Indices/StringIndices.rst

```diff
@@ -47,6 +47,21 @@ We will now show how we can create the different indices in SeqAn before we show
 All the mentioned indices belong to the generic :dox:`Index` class.
 A SeqAn index needs two pieces of information: the type of the :dox:`String` or :dox:`StringSet` to be indexed and the index specialization, such as :dox:`IndexEsa` or :dox:`FMIndex`.
 
+.. important::
+
+    Indices based on suffix arrays (also including the FM index) are built using secondary memory.
+    Building large indices can throw errors when you are running out of disk space or quota.
```

Comment:
Building large indices therefore can throw errors `,` when you are running... ? (comma)